interface/wx/intl.h

```diff
@@ -45,9 +61,16 @@ struct wxLanguageInfo
     wxUint32 WinLang, WinSublang;
     //@}
 
-    /// Human-readable name of the language.
+    /// Human-readable name of the language in English.
     wxString Description;
 
+    /**
+        Human-readable name of the language in native language.
```

Comment:
Another nitpick, but "native" might be a bit unclear here because it could be interpreted as meaning the current system language, so I'd change it to
```suggestion
        Human-readable name of the language in this language itself.
```

Reply:
I'm fine with the new wording.